src/libraries/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.Date.cs

```diff
@@ -42,103 +42,73 @@ public static void WriteDateTimeOffsetTrimmed(Span<byte> buffer, DateTimeOffset
         //   2017-06-12T05:30:45                  (interpreted as local time wrt to current time zone)
         public static void TrimDateTimeOffset(Span<byte> buffer, out int bytesWritten)
         {
+            const int maxLenNoOffset = JsonConstants.MaximumFormatDateTimeLength;
```

Comment:
```suggestion
            const int maxDateTimeLength= JsonConstants.MaximumFormatDateTimeLength;
```

Reply:
Actually if you do end up removing the unrolled logic it might make sense to remove these aliases altogether.